data/human/deep jobs.txt

```diff
@@ -353,3 +353,616 @@ mission "Record academic conference"
+	to offer
+		random < 50
+		has "deep security reveal"
+		"reputation: Deep Security" >= -50
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 50000 10000
+		"reputation: Deep Security" += 0.5
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+mission "Secure Transport Job (Basic) [1]"
+	name "Secure delivery to <planet>"
+	job
+	repeat
+	description "Deliver a secure package of supplies to <destination>. Payment is <payment>."
+	cargo "secure package" 1
+	to offer
+		random < 50
+		has "deep security reveal"
+		"reputation: Deep Security" >= -50
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 50000 10000
+		"reputation: Deep Security" += 0.5
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+mission "Secure Transport Job (Deep Security) [0]"
+	name "Deep Security delivery to <planet>"
+	job
+	repeat
+	description "Deliver a Deep Security package to <destination>. Payment is <payment>."
+	cargo "deep security package" 1
+	to offer
+		random < 20
+		has "deep security reveal"
+		"reputation: Deep Security" >= 100
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 80000 15000
+		"reputation: Deep Security" += 0.5
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+mission "Secure Transport Job (Deep Security) [1]"
+	name "Deep Security delivery to <planet>"
+	job
+	repeat
+	description "Deliver a Deep Security package to <destination>. Payment is <payment>."
+	cargo "deep security package" 1
+	to offer
+		random < 20
+		has "deep security reveal"
+		"reputation: Deep Security" >= 100
+	source
+		attributes "deep"
+		not attributes "station" "moon"
+	destination
+		attributes "deep"
+		not planet "Valhalla"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 80000 15000
+		"reputation: Deep Security" += 0.5
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
```

Comment:
What is the difference between these two missions? I'm rather unfamiliar with mission syntax so apologies if I'm missing something obvious here.

Reply:
Repeating jobs lets more than one instance of them appear on the boards at the same time.